.github/workflows/ci-backend.yml

```diff
@@ -155,84 +197,58 @@ jobs:
             - name: Add kafka host to /etc/hosts for kafka connectivity
               run: sudo echo "127.0.0.1 kafka" | sudo tee -a /etc/hosts
 
-            - name: Run posthog tests
+            - name: Set up needed files
               run: |
                   mkdir -p frontend/dist
                   touch frontend/dist/index.html
                   touch frontend/dist/layout.html
                   touch frontend/dist/shared_dashboard.html
-                  pytest -m "not ee" posthog/ --cov --cov-report=xml:coverage-postgres.xml
-            - name: Run EE tests
+
+            - name: Run FOSS tests
+              if: ${{ matrix.foss }}
+              run: |
+                  rm -rf ee/
+                  pytest -m "not ee" posthog/ --cov  --cov-report=xml:coverage-postgres.xml
```

Comment:
Are we sure we only want to run `posthog/` tests when no `ee` code is present? Could there be something in ee that breaks this down?

Reply:
I couldn't think of anything, this was mentioned as a cadveat in the description.